src/lib/mina/mina.ts

```diff
@@ -94,8 +96,8 @@ setActiveInstance({
   },
 });
 
-const Transaction = {
-  fromJSON(json: Types.Json.ZkappCommand): Transaction {
+const TransactionUtil = {
```

Comment:
Was this necessary? Would prefer to leave it the same name

Reply:
I think it's exported as Mina.Transaction, and people use it

I always like to keep the number of exported names small, and use this pattern a lot

But I'm interested in the auto importing issue, what do you mean?

Reply:
It's also the same as if Transaction were a class - then the Transaction name would serve as a type as well as a collection of static methods as well as a constructor

Reply:
I often use auto-imports so that I don't have to manually import symbols. I just use the symbol, and the import statement is generated. This poses some risk / it's easy to accidentally import from the wrong source. But it's a productivity gain imo. Having two `Transaction` symbols in separate files poses confusion about autoimport source.

https://github.com/o1-labs/o1js/assets/4893548/d424bb0a-6810-4634-9c18-47fbe43f08a4

To get around this––while keeping the shadowing––I've moved the runtime "shadow" into `lib/mina/transaction.ts`. Users can still make use of the type-only import if they'd like. Meanwhile `Transaction` is also a namespace, off of which the `fromJSON` method can be accessed.

> Sidenote: shall we rename `fromJSON` to `fromJson`? The `"JSON"` piece is––although an abbreviation––a common term, and thus can be treated as a full segment of the camelcasing.

Reply:
Got it, thanks for moving it (I also use auto imports of course :D)

Json would be more logical but I think JSON might look prettier / more familiar. In any case but something I want to change 5 min before v1